src/main/java/org/spongepowered/api/statistic/achievement/Achievement.java

```diff
@@ -24,34 +24,19 @@
  */
 package org.spongepowered.api.statistic.achievement;
 
-import org.spongepowered.api.CatalogType;
-import org.spongepowered.api.Sponge;
+import org.spongepowered.api.item.inventory.ItemStack;
 import org.spongepowered.api.statistic.Statistic;
-import org.spongepowered.api.text.translation.Translatable;
 import org.spongepowered.api.text.translation.Translation;
-import org.spongepowered.api.util.ResettableBuilder;
 import org.spongepowered.api.util.annotation.CatalogedBy;
 
 import java.util.Collection;
 import java.util.Optional;
 
-import javax.annotation.Nullable;
-
 /**
  * Represents an in-game achievement which may be earned by or given to players.
  */
 @CatalogedBy(Achievements.class)
-public interface Achievement extends CatalogType, Translatable {
-
-    /**
-     * Creates a new {@link Builder} to build a {@link Achievement}.
-     *
-     * @return The new builder
-     */
-    static Builder builder() {
-        return Sponge.getRegistry().createBuilder(Builder.class);
-    }
-
+public interface Achievement extends Statistic {
 
```

Comment:
I originally decided against this as it for a number of reasons. Primarily that the actual statistics value is pretty useless for achievements and representing it as such is a bit of a hack. I'm guessing you made this change for ease of implementation?